app/components/UI/FiatOnRampAggregator/components/PaymentOption.tsx

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-require-imports */
+import React from 'react';
+import { StyleProp, StyleSheet, TextStyle, View } from 'react-native';
+import Box from './Box';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import Feather from 'react-native-vector-icons/Feather';
+import MaterialsCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import MaterialsIcons from 'react-native-vector-icons/MaterialIcons';
+import { colors } from '../../../../styles/common';
+import { Image } from 'react-native-animatable';
+import CustomText from '../../../Base/Text';
+import BaseListItem from '../../../Base/ListItem';
+
+const visa = require('./images/visa.png');
+const sepa = require('./images/sepa.png');
+const mastercard = require('./images/mastercard.png');
+
```

Comment:
Remove top eslint disable and disable the block only where it is used.

```suggestion
/* eslint-disable import/no-commonjs, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
const visa = require('./images/visa.png');
const sepa = require('./images/sepa.png');
const mastercard = require('./images/mastercard.png');
/* eslint-enable import/no-commonjs, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */

```